js/radio/shared/Radio.svelte

```diff
@@ -10,8 +10,19 @@
 	export let selected: string | null = null;
 
 	const dispatch = createEventDispatcher<{ input: string | number }>();
+	let is_selected = false;
```

Comment:
Is this used anywhere? Looks like we can bring it into the function completely.

Reply:
To apply css for the selected class!

Reply:
ah cool